.github/workflows/checks.yml

```diff
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-18.04
     name: check tests
     steps:
-    - uses: actions/checkout@v2
```

Comment:
@coffemakingtoaster It's preferable to reference a release version to ensure we don't pull in breaking changes unexpectedly. It looks like `actions/checkout@v3` is the latest. Can we update to use those instead?